xrpl/cli-utils.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { Option } = require('commander');
+const { addEnvOption } = require('../common/cli-utils');
+
+const addBaseOptions = (program, _options = {}) => {
+    addEnvOption(program);
+
+    program.addOption(
+        new Option('-n, --chainName <chainName>', 'chain to run the script over').makeOptionMandatory(true).env('CHAIN'),
+    );
+
+    program.addOption(new Option('-s, --seed <seed>', 'seed used to derive wallet keypair').makeOptionMandatory(true).env('SEED'));
```

Comment:
```suggestion
        program.addOption(new Option('-p, --privateKey <privateKey>', 'private key').makeOptionMandatory(true).env('PRIVATE_KEY'));

        program.addOption(
            new Option('--privateKeyType <privateKeyType>', 'private key type')
                .makeOptionMandatory(true)
                .choices(['seed'])
                .default('seed')
                .env('PRIVATE_KEY_TYPE'),
        );
```
perhaps for consistency